sdk/servicebus/azure-servicebus/azure/servicebus/aio/_transport/_base_async.py

```diff
@@ -134,7 +149,7 @@ async def send_messages_async(sender, message, logger, timeout, last_exception):
 
     @staticmethod
     @abstractmethod
-    def create_source(source, offset, selector):
+    def create_source(source, session_filter):
```

Comment:
the params was changed here but not in the docstring. Wont this affect implementor of the ABC ?

Reply:
ah forgot to update the docstring, thanks!

but the ABC actually only checks the method signatures + mypy checks typehints, but having different docstring won't raise any mypy/other errors.